tests/foreman/api/test_organization.py

```diff
@@ -283,13 +268,12 @@ def test_associate_with_subnet(self):
 
     @tier2
     @skip_if_bug_open('bugzilla', 1230865)
-    def test_associate_with_media(self):
+    def test_positive_update_media(self):
```

Comment:
Maybe `test_positive_add_media` as it is actually adding rather than updating(changing)